sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -167,6 +168,28 @@ pi_result piPluginGetLastError(char **message) {
   return ErrorMessageCode;
 }
 
+// Optimization strings.
+static const char *EmptyStr = "";
+
+// Returns plugin specific backend option.
+// Current support is only for optimization options.
+// Return empty string for esimd emulator.
+// TODO: Determine correct string to be passed.
+pi_result piPluginGetBackendOption(pi_platform platform,
+                                   const char *frontend_option,
+                                   const char **backend_option) {
+  using namespace std::literals;
+  if (frontend_option == nullptr)
+    return PI_ERROR_INVALID_VALUE;
+  if (frontend_option == "-O0"sv || frontend_option == "-O1"sv ||
+      frontend_option == "-O2"sv || frontend_option == "-O3"sv ||
+      frontend_option == ""sv)) {
+    *backend_option = EmptyStr;
```

Comment:
why empty?

Reply:
Same as above answer. Main focus is on opencl and level_zero. i have added placeholders for other backends which can be modified at a later stage.

Thanks